cmd/operator-sdk/test/local.go

```diff
@@ -206,7 +212,11 @@ func testLocalGoFunc(cmd *cobra.Command, args []string) error {
 		testArgs = append(testArgs, strings.Split(tlConfig.goTestFlags, " ")...)
 	}
 	if tlConfig.namespace != "" || tlConfig.noSetup {
-		testArgs = append(testArgs, "-"+test.SingleNamespaceFlag, "-parallel=1")
+		// check for WATCH_NAMESPACE="" in --up-local
```

Comment:
IMHO it could be simplified. If the user does not inform the namespace then it will be as '" by default. 
WDYT @joelanford and @AlexNPavel ?

Reply:
true, but it can look like magic.
In addition, in `operator-sdk up local`, `WATCH_NAMESPACE=""` is set to "" only if specified explicitly. 

So if we make `"" ` default for `operator-sdk test --up-local`, it might break expectations

Reply:
IMHO: It is quite intuitive assume that if no namespace was specified it will valid for all. Also, it will keep simple the implementation as well and reduce its complexity. 

However,  please wait for the other inputs as well.

Reply:
ref: https://github.com/operator-framework/operator-sdk/blob/082e0468f89b6496be0791129be5b32c6d28d959/cmd/operator-sdk/up/local.go#L84